lib/algo/NonuniformDMD.cpp

```diff
@@ -20,7 +20,8 @@ NonuniformDMD::NonuniformDMD(int dim, Vector* state_offset,
                              Vector* derivative_offset) :
     DMD(dim, state_offset)
 {
-    d_derivative_offset = derivative_offset;
+    //setOffset(state_offset, 0);
```

Comment:
Should this commented line be kept? There is another line like it below.

Reply:
This comment is now changed.